### DISCUSSION TITLE
Ownership and Attribution of Assets on VDN Portal

### DISCUSSION THREAD

**Discussion by Prerna789** [Sunbird-coKreat/Community#6] [General Discussions]
Hi @vinukumar-vs @AmiableAnil

Wanted to get an understanding of how Knowlg APIs handle ownership and attribution of assets? 
Context here is that we want to allow Contribution and Sourcing Org Reviewers to edit content on VDN portal.
So want to understand how ownership is managed currently at the backend and the implications of this functionality on the APIs

Thanks!
cc - @Krishnaj20 @manojSRawat

**Comment by pramodkvarma** [comment]
Content "ownership" and "attributions" (of all nature) are simply meta attributes. "update" control is what you are asking (not legal/IP ownership), I presume. As I understand, "update control" is with the "creator user+org". @vinukumar-vs or @vrayulu can confirm.

**Reply by vinukumar-vs** [reply]
The "owner" & "attributions" are part of meta attributes itself. But SB-Knowlg doesn't have any logic or behavior on these attributes.

@Prerna789  Can you please elaborate more on "ownership"? What are you expecting as part of asset ownership?

If you are looking for asset ownership as a role or creator who can update the asset, this logic is handled on [sb-knowledge-mw](https://github.com/project-sunbird/knowledge-mw-service). This is being handled as part of Sunbird-Portal integration(API's will always go through knowledge-mw for any asset CRUD operations).

@pallakartheekreddy can share more details on how it is handled as part of co-create.
